x-pack/plugins/observability/public/mock/uptime.mock.ts

```diff
@@ -0,0 +1,1228 @@
+/*
```

Comment:
Is this still needed?

Reply:
I moved all mocks to `pages/overview/` they will be used to show the storybook.